labels/en-US.json

```diff
@@ -630,6 +630,8 @@
   "quoteExpiredBuyerBlob2": "Please submit a new request or contact your sales representative for help",
   "quoteExpiredSeller": "Hi, <br /><br /><br /> Quote \"{0}\" (#{1}) has expired as of",
   "quoteHistory": "Quote History",
+  "quoteShippingMethods": "Shipping Methods",
```

Comment:
same keys should be in alphabetical order

Reply:
done